config/legacyconfig/config.go

```diff
@@ -199,6 +252,7 @@ type SyncNodes struct {
 }
 
 type SyncGeneric struct {
+	RBACValues
```

Comment:
Had to add this in order for the unit test (`generic sync example`) for 0.19 configs to work with the strict decoding with `DisallowUnknownFields`.

Reply:
Same as above, potentially use a custom Unmarshaller.

Reply:
Same as above its fine as it is

Reply:
As discussed. Idea was to inline the fields under rbac directly.